signature-pad/build.gradle

```diff
@@ -3,25 +3,24 @@ apply plugin: 'com.github.dcendents.android-maven'
 apply plugin: 'com.jfrog.bintray'
 
 dependencies {
-
 }
 
 Properties properties = new Properties()
 final def bintrayPropertiesFile = project.rootProject.file('bintray.properties')
 if (bintrayPropertiesFile.exists()) properties.load(bintrayPropertiesFile.newDataInputStream())
 
 android {
-    compileSdkVersion Integer.parseInt(project.ANDROID_BUILD_SDK_VERSION)
-    buildToolsVersion project.ANDROID_BUILD_TOOLS_VERSION
+    compileSdkVersion 27
 
     defaultConfig {
-        minSdkVersion 9
+        minSdkVersion 15
```

Comment:
Why this bump in minimum supported version? If it's because of the `.getPressure()` feature, can't we check at runtime which SDK version is running on the customer device and fallback to `1.0` in case `.getPressure()` method is not available?